almanac.go

```diff
@@ -6,9 +6,9 @@ import (
 )
 
 // AlmanacNetworkSearch performs a call to almanac.network.search
-func (c *Conn) AlmanacNetworkSearch () (*responses.SearchResponse, error) {
+func (c *Conn) AlmanacNetworkSearch (req requests.Request) (*responses.SearchResponse, error) {
```

Comment:
should this be a `requests.SearchRequest`?

Reply:
Yep - updated. Thanks. Guess when you want an empty request, doesn't matter too much which one you send.